packages/dashboard/src/components/resourceExplorer/stencil/index.tsx

```diff
@@ -14,6 +14,14 @@ interface StencilResourceExplorerProps {
   treeQuery: SiteWiseQuery['assetTree']['fromRoot'];
 }
 
+const DEFAULT_COLUMNS = [
```

Comment:
can't use type `TableProps.ColumnDefintion<AssetSummary>[]` here. Because resource explorer use `@awsui`. `ColumnDefintion` in `@cloudescape` and `@awsui` are slightly mismatched.